packages/rtk-codemods/transforms/enhanceEndpoint/test.js

```diff
@@ -0,0 +1,9 @@
+'use strict'
+
+const { runTransformTest } = require('codemod-cli')
+
+runTransformTest({
+  name: 'enhanceEndpoint',
+  path: require.resolve('./index.ts'),
+  fixtureDir: `${__dirname}/__testfixtures__/`
+})
```

Comment:
```suggestion
import path from 'node:path'
import { runTransformTest } from '../../transformTestUtils.js'
import transform, { parser } from './index.js'

runTransformTest(
  'enhanceEndpoints',
  transform,
  parser,
  path.join(__dirname, '__testfixtures__')
)
```

We should probably get rid of [**`codemod-cli`**](https://github.com/rwjblue/codemod-cli#readme) since it hasn't been updated in about 4 years and does not work with later versions of TypeScript.